src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/ServiceHubContextFactory.cs

```diff
@@ -1,33 +1,35 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.SignalR.Management
 {
     internal class ServiceHubContextFactory
```

Comment:
This class is not needed anymore. To minimize changes in this PR, this class is not removed at this time.